packages/react-avatar/src/components/Avatar/useAvatar.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
 import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utils';
-import { AvatarProps, defaultAvatarSize } from './Avatar.types';
+import { AvatarProps, AvatarState, defaultAvatarSize } from './Avatar.types';
 import { useMergedRefs } from '@fluentui/react-hooks';
 import { getInitials as defaultGetInitials, nullRender, assertNever } from '@fluentui/utilities';
 import { Image } from '../Image/index';
 import { ContactIcon as DefaultAvatarIcon } from '@fluentui/react-icons-mdl2';
 
-export const avatarShorthandProps: (keyof AvatarProps)[] = ['label', 'image', 'badge'];
+export const avatarShorthandProps = ['label', 'image', 'badge'];
```

Comment:
(nit) why remove the type on this?

Reply:
It was a typing issue, now it's fixed, restored 👍